mpi-proxy-split/lower-half/static_libs.txt

```diff
@@ -1 +1 @@
--lpmi -lpmi2 /opt/cray/libfabric/1.15.2.0/lib64/libfabric.a -lcxi -L/global/homes/z/zz217/local/lib -L/global/cfs/cdirs/cr/pm_dependencies -ljson-c -lcurl  -lssl -lcrypto -lz -latomic -lpthread /opt/cray/pe/pals/1.2.11/lib/libpals.a libnl-3.a
+-lpmi -lpmi2 /opt/cray/libfabric/1.15.2.0/lib64/libfabric.a -lcxi -L/global/homes/z/zz217/local/lib -L/global/cfs/cdirs/cr/pm_dependencies -ljson-c -lcurl  -lssl -lcrypto -lz -latomic -lpthread /opt/cray/pe/pals/1.2.11/lib/libpals.a libnl-3.a -ljansson -Xlinker -zmuldefs
```

Comment:
Please add to the Makefile near static_libs.txt a comment:
     `# -zmuldefs used in static_libs.txt because libjansson.a and libjson.a both want to define [FILL IN}.`
     `# We prefer libjansson.a`

Reply:
Good point. Done. Thanks Gene!